src/routes/healthcheck/healthcheck-get.ts

```diff
@@ -1,19 +1,16 @@
 import { Request, Response } from "express";
 import { getLogger } from "config/logger";
-import { envVars } from "config/env";
 import { deepCheckCryptor } from "./deep-check-cryptor";
 
 const logger = getLogger("healthcheck");
 export const HealthcheckGet = async (_: Request, res: Response): Promise<void> => {
 
-	if (envVars.MICROS_GROUP === "Worker") {
```

Comment:
Webserver healtcheck will be hit very often, like every minute. Have you considered this?

Reply:
Are you suggesting that the cryptor sidecar can't handle 2 requests per minute?

Reply:
It should be able to handle that, all good.